### DISCUSSION TITLE
Mac OS 11 Big Sur and below is not supported

### DISCUSSION THREAD

**Discussion by vksolanki-planetsolanki** [phcode-dev/phoenix#1579] [Q&A] [answered]
Hi All,

When I launch Pheonix Code I am presented with a dialog that reads, "Oops. Phoenix could not be started due to missing file system library."  The only button to click on is OK.

<img width="294" alt="image" src="https://github.com/phcode-dev/phoenix/assets/168394798/7846b0d8-e01d-46a9-8e48-80d3dd92f0ad">

MacOS 11.7.2 Big Sur
Intel Core i7

Any help appreciated.

Thanks - V

**Comment by abose** [comment]
Related to https://github.com/phcode-dev/phoenix/issues/1565

**Comment by abose** [comment, marked as answer]
Thanks for reporting the issue @vksolanki-planetsolanki 
Phoenix Code requires `macOS 12 Monterey` or higher because Apple introduced some system libraries starting with macOS 12, which Phoenix Code relies on. Apple doesnt seem to be patching `macOS Big Sur` with those updates (See image below). See related issue for more details. 

![image](https://github.com/phcode-dev/phoenix/assets/5336369/1cde8b1f-6f09-4ffb-aae4-3570af257935)
